src/cljs/org/broadinstitute/firecloud_ui/common/modal.cljs

```diff
@@ -18,14 +19,26 @@
   ;; Forces create-element so the caller can refer to refs in the dialog.
   (react/call :push-modal @instance (react/create-element child)))
 
-
 (declare OKCancelForm)
-
-
 (defn push-ok-cancel-modal [props]
   (react/call :push-modal @instance (react/create-element OKCancelForm props)))
 
 
+(defn- push-error [content]
+  (push-ok-cancel-modal
+    {:header [:div {:style {:display "flex" :alignItems "center"}}
+              (icons/icon {:style {:color (:exception-red style/colors)
+                                   :marginRight "0.5em"}} :error)
+              "Error"]
+     :content [:div {:style {:maxWidth "50vw"}} content]
```

Comment:
I would suggest an absolute pixel value for maxWith, since this doesn't really work on mobile. No need to change, since we're definitely not supporting mobile yet, but just noting the pattern.